test/extension/commandExecutor.test.ts

```diff
@@ -125,5 +128,79 @@ suite("commandExecutor", function() {
                     assert.fail(null, null, "react-native command was not expected to fail");
                 });
         });
+
+        test("getReactNativeVersion should return version string if there is 'react-native' field in project package.json", (done: MochaDone) => {
```

Comment:
I think we can create a separate suite for that and add setup and teardown functions there to avoid code duplicating.